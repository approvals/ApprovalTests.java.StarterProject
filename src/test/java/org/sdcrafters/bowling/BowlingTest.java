package org.sdcrafters.bowling;

import org.approvaltests.Approvals;
import org.junit.Test;

public class BowlingTest {

	@Test
	public void test() {
		// create a bowl game
		Bowling b = new Bowling();
		// roll 1,1,2,2,3,3,4,4,5,5,3,2,10,3,2,10,10,10,10
		b.roll(1, 1, 2, 2, 3, 3, 4, 4, 5, 5, 3, 2, 10, 3, 2, 10, 10, 10, 10);
		// verify score
		Approvals.verify(b.toString());
	}

}
