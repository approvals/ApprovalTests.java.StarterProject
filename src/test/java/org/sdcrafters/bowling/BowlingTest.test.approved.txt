1) 1,1 [2] = 2
2) 2,2 [4] = 6
3) 3,3 [6] = 12
4) 4,4 [8] = 20
5) 5,5 [13] = 33
6) 3,2 [5] = 38
7) 10  [15] = 53
8) 3,2 [5] = 58
